### PR TITLE
Added the getTitle and getDescription Functions in FieldsExtractorEngine Class

### PR DESCRIPTION
**Pull request by real-Vansh-Khare**
The getTitle function extracts the title of a given question DOM node.
The getDescription function extracts the description of a given question DOM node based on the DOM tree.
These functions seem to work for all types of questions.

### CODE REVIEW

**Comment by rootCircle** [comment]
Your code also catches the extra `*` for fields that are mandatory, its purpose serves no part of question and can poison the question itself.

Can you fix that?

File: filler.js [hunk 1]
```diff
@@ -578,6 +581,37 @@ class FieldsExtractorEngine {
...
 
 	}
+
+	// Testing on Different Forms required
+	getTitle(element) {
+		// Extracts the title of the question
+		// The extraction is based on the DOM tree
+		// The required node is obtained by going into the first child element 5 times
+		// Return Type : String
+		let required = element.children[0]
```
Comment (rootCircle) [inline filler.js]: Why not try accessing the heading, as it is contained in a div with role attribute = heading?
Title is probably in the first span inside that heading.
And description might be in second child to parent of that div.

hint : try element.querySelector("div[role=heading]")

Comment (real-Vansh-Khare) [reply filler.js]: Removed * from question title and used div[role=heading] for faster dom traversal

**Comment by rootCircle** [comment]
Did some changes, now getDescription() return null rather than err string to remove ambiguity
Rest good

**Comment by rootCircle** [comment]
Congratulations for your first PR in this repo!
Merging now

**Pull request merged**